lib/bundler.rb

```diff
@@ -177,16 +176,12 @@ def user_home
       end
     end
 
-    def tmp_home_path(login, warning)
-      login ||= "unknown"
+    def tmp_home_path(_login, warning)
```

Comment:
I assume you didn't change the signature to remove the `login` parameter for backwards compatibility?

I don't think anybody should be using this method, and it's returning an insecure path anyways, so I would vote for making it private and removing the now unused parameter.

Thoughts?